snippets/consumers.json

```diff
@@ -0,0 +1,25 @@
+{
+    "consumer": {
+        "prefix": [
+            "consumer"
+        ],
+        "body": [
+            "CONSUMER ${1:consumer-group-id}",
+            "topic: ${2:topic_name}",
+            "from: ${3|earliest,latest,0,1,2,3,4,5|}"
```

Comment:
makes no sense to list ints. Just put 0, so we know numbers are supported

Reply:
I wanted just higlight that from could be an integer. Makes sense to have just 0.

fixed